### PR TITLE
Merge the two command line interface

### PR DESCRIPTION
**Pull request by rjaccard**

### CODE REVIEW

File: be1-go/cli/mod.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Package organizer contains the entry point for starting the organizer
+// Package cli contains the entry point for starting the organizer
 // server.
```
Comment (Anghul) [inline be1-go/cli/mod.go:2]: You don't mention the witness in this comment

File: be1-go/cli/mod.go [hunk 2]
```diff
@@ -19,23 +19,30 @@ import (
...
+
+	// get command line args which specify public key, port to use for clients
+	// and witnesses, witness' address
 	clientPort := cliCtx.Int("client-port")
 	witnessPort := cliCtx.Int("witness-port")
 	if clientPort == witnessPort {
 		return xerrors.Errorf("client and witness ports must be different")
 	}
-	witness := cliCtx.StringSlice("other-witness")
+	otherWitness := cliCtx.StringSlice("other-witness")
```
Comment (Anghul) [inline be1-go/cli/mod.go:41]: I think we would need a variable to add one (or more) organizer too

**Review by Anghul — CHANGES_REQUESTED** [review]
Good job. I think that you could remove the package cli/utility and put back its content in the cli as I created it only because I had to use a function in both the witness and the organizer's cli

File: be1-go/cli/mod.go (hunk 2 above)
Comment (rjaccard) [reply be1-go/cli/mod.go:43]: I will leave it for some future work as the system does not support multiple organisers yet

**Review by nkcr — APPROVED** [review]
Good for me 👍 
Sonarcloud is not happy because there is less than 20% coverage on new code :/

**Review by Anghul — APPROVED** [review]

**Pull request merged**